float32 x1
float32 y1
float32 yaw1

float32 x2
float32 y2
float32 yaw2

float32 timestamp
float32 velocity1
float32 velocity2
